planner/optimizer/.gitignore

```diff
@@ -4,5 +4,4 @@ target/
 .classpath
 .project
 .metadata
-.settings/
-src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/output*
```

Comment:
They stay ignored because everything under target/ is ignored too. It looked like the folder "resources" contained 2 different kind of files, input files and generated ones.